### PR TITLE
TOMEE-2486 | Translate to Portuguese: examples/mp-config-example

### PR DESCRIPTION
**Pull request by rhuan080**
Signed-off-by: rhuanrocha <rhuan080@gmail.com>

### CODE REVIEW

File: examples/mp-config-example/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+3 . Para pegar a propriedade do Config com getValue chamar:
+
+----
+GET http://localhost:8080/mp-config-example/sample/javaVersion
+----
+
+== Características de Config
+
+
+MicroProfile Confi é uma solução para externalizar configurações de microsserviços.
```
Comment (Daniel-Dos) [inline examples/mp-config-example/README_pt.adoc]: small fix - "Confi" to "Config"

File: examples/mp-config-example/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Você pode também inserir um valor padrão para ele, caso o config não encontre a propriedade no contexto ele irá usar o valor padrão
+
+[source,java,numbered]
+----
+@Inject
+@ConfigProperty(name = "defaultProperty", defaultValue = "ALOHA")
+private String defaultProperty;
+----
+
+O objecto config também pode ser injetado. Então use o método getValue() para recuperar a propriedade individual.
```
Comment (Daniel-Dos) [inline examples/mp-config-example/README_pt.adoc]: here too  "objecto" to "objeto"

File: examples/mp-config-example/README_pt.adoc (hunk 1 above)
Comment (rhuan080) [reply examples/mp-config-example/README_pt.adoc]: Thank you!

File: examples/mp-config-example/README_pt.adoc (hunk 2 above)
Comment (rhuan080) [reply examples/mp-config-example/README_pt.adoc]: Thank you!

**Comment by Daniel-Dos** [comment]
It's very good for me. 

thanks @rhuan080  .

File: examples/mp-config-example/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+== Rodando a aplicação:
+
+[source,bash]
+----
+mvn clean install tomee:run 
+----
+
+== API
+
+Na aplicação, tem 3 formas de injetar valores usando config
```
Comment (otaviojava) [inline examples/mp-config-example/README_pt.adoc]: Acredito que a virgula não seja necessário aqui.
E talvez substituir `tem` por `existem`
 Na aplicação existem três formas de injetar valores usando config

**Comment by cesarhernandezgt** [comment]
Hi @rhuan080, thanks for the PR :)!

I created the JIRA ticket: https://issues.apache.org/jira/browse/TOMEE-2486, usually we create first the JIRA, start a email thread on the mailing list to:

1. Request to be assigned to the JIRA, this will allow visibility and prevent duplication of work.
2. Send status when your PR is ready to be reviewed.

Can you please share your JIRA username so I can assign the ticket TOMEE-2486 to you?

For further reference you can check: http://tomee.apache.org/community/contributing/workflow.html

File: examples/mp-config-example/README_pt.adoc (hunk 3 above)
Comment (rhuan080) [reply examples/mp-config-example/README_pt.adoc]: @otaviojava fiz a alteração conforme sugerido!

**Comment by rhuan080** [comment]
Hi @cesarhernandezgt , thank you!

My username is **rhuanrocha**.

**Comment by cesarhernandezgt** [comment]
@rhuan080 Can we continue this conversation in the mailing list thread with subject "TOMEE-2486 | Translate to Portuguese: examples/mp-config-example" ?
Remember that "if didn't happen in the mailing list, it didn't happen" ;).
I can't find your username rhuanrocha in https://issues.apache.org/jira

**Comment by cesarhernandezgt** [comment]
Hi @rhuan080 
Thank you for your contribution, in order to move forward this PR please provide the update on the email thread "TOMEE-2486 | Translate to Portuguese: examples/mp-config-example"  on the mailing list to make reference to both the Jira and the PR. I was able now to find your JIRA id.

In this way, we collaborate with the committers to move forward the contributions and follow the Apache Way "If it didn't happen on a mailing list, it didn't happen." [1] along with the recommended Apache TomEE Workflow [2].

[1] https://community.apache.org/newbiefaq.html#NewbieFAQ-IsthereaCodeofConductforApacheprojects?
[2] http://tomee.apache.org/community/contributing/workflow.html

**Pull request merged**